app/assets/stylesheets/application.scss

```diff
@@ -1,10 +1,21 @@
 @import "bootstrap/scss/bootstrap";
 @import "./base/variables.scss";
 @import "./components/login";
-
+@import "./components/signup";
  html, body {
   background-color: $background_color;
   font-family: $poppins;
   margin: 0;
   width: 100%;
 }
+
+html {
+  font-size: 62.5%;
+  box-sizing: border-box;
+  scroll-padding-top: 0rem;
+}
```

Comment:
What is the reason to have two html label styles?

Reply:
done